test/python/fe_api/grouped_gemm/test_discrete_grouped_gemm_dswiglu_jax.py

```
@@ -0,0 +1,186 @@
+        rtol=2e-5,
+        atol=1e-4,
+        err_msg="dswiglu dprob_tensor: JAX output differs from torch output beyond atomic-ordering tolerance",
+    )
+
+
+@pytest.mark.L0
+def test_discrete_grouped_gemm_dswiglu_jax_errors():
+    skip_unless_sm100()
+    from cudnn import discrete_grouped_gemm_dswiglu_wrapper_sm100
+
+    a_np, b_np, c_np, sfa_u8, sfb_u8, offsets_np, alpha_np, beta_np, prob_np, norm_const_np = make_problem()
```

Ruff (0.16.1)

[warning] Unpacked variable `a_np` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)